dspace-server-webapp/src/main/java/org/dspace/app/rest/model/ClarinLicenseLabelExtendedMappingRest.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.dspace.app.rest.RestResourceController;
+import org.dspace.content.clarin.ClarinLicense;
+
```

Comment:
add doc

Reply:
This class should be unuseful. I delete it.